### ISSUE TITLE
lodash vulnerability

### ISSUE THREAD

**Issue by danielrozenberg** [craftzdog/extract-main-text-node#2]
I'm getting a warning from `npm audit` when installing this module:

```
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ extract-main-text                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ extract-main-text > lodash                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Would it be possible to update the version of lodash used by this module?

**Comment by craftzdog** [comment]
okay

**Comment by craftzdog** [comment]
published `extract-main-text@1.0.3` with no vulnerabilities.
Thanks for reporting!
